app/pkg/email/smtp/smtp.go

```diff
@@ -157,3 +168,15 @@ func sendMail(localName, serverAddress string, a gosmtp.Auth, from string, to []
 	}
 	return c.Quit()
 }
+
+func generateMessageID() (string, error) {
+	timestamp := strconv.FormatInt(time.Now().UTC().UnixNano(), 10)
+	unq := uuid.New()
+	randStr := unq.String()
+	host, err := os.Hostname()
```

Comment:
We shouldn't use os.Hostname here, the hostname is usually completely different than the real domain name. There's a `localname` variable that we could use instead.